src/cli/flags/s3.go

```diff
@@ -55,8 +54,7 @@ func S3FlagOverrides(cmd *cobra.Command) map[string]string {
 
 func PopulateS3Flags(flagset PopulatedFlags) map[string]string {
 	s3Overrides := make(map[string]string)
-	// TODO(pandeyabs): Move account overrides out of s3 flags
-	s3Overrides[account.AccountProviderTypeKey] = account.ProviderM365.String()
+
 	s3Overrides[storage.StorageProviderTypeKey] = storage.ProviderS3.String()
```

Comment:
I can't include line 56 in this, but this can be shortened ot a single line:
```suggestion
	s3Overrides = map[string]string{storage.StorageProviderTypeKey: storage.ProviderS3.String()}
```